crates/ln-dlc-node/src/tests/just_in_time_channel.rs

```diff
@@ -120,43 +176,16 @@ async fn just_in_time_channel() {
 
     assert_eq!(
         coordinator_balance_after.available - coordinator_balance_before.available,
-        JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT + flat_routing_fee + liquidity_routing_fee
+        coordinator_balance_before_payment + flat_routing_fee + liquidity_routing_fee
     );
 
     assert_eq!(
         payee_balance_after.available - payee_balance_before.available,
         invoice_amount
     );
-}
-
-/// Used to ascertain if a payment will be routed through a channel
-/// according to the
-/// `max_inbound_htlc_value_in_flight_percent_of_channel`
-/// configuration flag of the receiving end of the channel.
-fn does_inbound_htlc_fit_as_percent_of_channel(
-    receiving_node: &Node,
-    channel_id: &[u8; 32],
-    htlc_amount_sat: u64,
-) -> Result<bool> {
-    let htlc_amount_sat = Decimal::from(htlc_amount_sat);
-
-    let max_inbound_htlc_as_percent_of_channel = Decimal::from(
-        receiving_node
-            .user_config
-            .channel_handshake_config
-            .max_inbound_htlc_value_in_flight_percent_of_channel,
-    );
 
-    let channel_size_sat = receiving_node
-        .channel_manager
-        .list_channels()
-        .iter()
-        .find_map(|c| (&c.channel_id == channel_id).then_some(c.channel_value_satoshis))
-        .context("No matching channel")?;
-    let channel_size_sat = Decimal::from(channel_size_sat);
+    // no additional just-in-time channel should be created.
+    assert_eq!(coordinator.channel_manager.list_channels().len(), 2);
```

Comment:
🙃 We could move this to the test since we don't actually know if the caller's `coordinator` should only have 2 channels.